bindata/v4.1.0/kube-apiserver/pod.yaml

```diff
@@ -38,6 +38,11 @@ spec:
     command: ["/bin/bash", "-ec"]
```

Comment:
"-euxo" , "pipefail", "-c"?

Reply:
it's preexisting. Don't want to change more than necessary.